packages/icons/src/Icon.tsx

```diff
@@ -86,4 +86,7 @@ export const Icon = forwardRef<SVGSVGElement, IconProps>((props, ref) => {
     );
 });
 
-Icon.displayName = "Icon";
+const _Icon = slot("avatar", Icon);
```

Comment:
[Question] Is it called "avatar" on purpose?

Reply:
no :(